src/EventListener/ExceptionListener.php

```diff
@@ -37,4 +55,35 @@ public function onKernelException(GetResponseForExceptionEvent $event)
 
         parent::onKernelException($event);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function logException(\Exception $exception, $message)
```

Comment:
You can typehint `$message` with `string`

Reply:
```
Declaration of ApiPlatform\Core\EventListener\ExceptionListener::logException(Exception $exception, string $message) should be compatible with Symfony\Component\HttpKernel\EventListener\ExceptionListener::logException(Exception $exception, $message)
```
cant do that